.github/dependabot.yaml

```diff
@@ -0,0 +1,39 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/drivers/
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/jobs/apollo-src/
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/jobs/integration-tests/
```

Comment:
The apollo-ci images on these dockerfiles need to be manually mirrored into OSCI, so auto-updating these are kind of pointless on their own (and we are also moving to use GHA soon, so the .openshift-ci will cease to exist soon).